modules/mod_authentication/controllers/controller_logon.erl

```diff
@@ -489,4 +492,13 @@ get_by_reminder_secret(Code, Context) ->
     end.
 
 
+%% @doc Get actions that should be wired after successful logon
+-spec get_post_logon_actions(list(), #context{}) -> list().
+get_post_logon_actions(WireArgs, Context) ->
+    case z_notifier:foldl(#logon_actions{args=WireArgs}, [], Context) of
+        [] ->
+            [{redirect, [{location, cleanup_url(get_ready_page(Context))}]}];
```

Comment:
Returning the default here instead of in an `observe_logon_actions` because I think if you want custom actions, you probably don’t want the redirect as well (it will be included because of the `foldl`).